googlepubsub/src/main/scala/akka/stream/alpakka/googlepubsub/scaladsl/GooglePubSub.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.googlepubsub.scaladsl
+
+import java.security.PrivateKey
+
+import akka.actor.ActorSystem
+import akka.stream.Materializer
+import akka.stream.alpakka.googlepubsub._
+import akka.stream.scaladsl.{Flow, Keep, Sink, Source}
+import akka.{Done, NotUsed}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object GooglePubSub extends GooglePubSub {
+  private[googlepubsub] override val httpApi = HttpApi
```

Comment:
This is for testing with a mock `HttpApi` in `GoogleCloudPubSubSpec`. Is there a better way ?